tests/components/Sidebar-cy.js

```diff
@@ -16,8 +16,7 @@ describe("Sidebar", function() {
     it("opens nested items in the sidebar when clicked", function() {
```

Comment:
These tests were removed to accommodate the changes in the sidebar.
They are re-worked in this PR where sidebar toggle is added back https://github.com/dcos/dcos-ui/pull/3097